src/Components/Publishing/Nav/Nav.tsx

```diff
@@ -79,23 +70,29 @@ export class NavComponent extends React.Component<Props, State> {
   }
 }
 
-const Div: StyledFunction<DivProps & React.HTMLProps<HTMLDivElement>> = styled.div
+const Div: StyledFunction<DivProps & React.HTMLProps<HTMLDivElement>> =
+  styled.div
 
 const NavContainer = Div`
-  background-color: ${props => props.transparent ? "transparent" : "black"};
+  background-color: ${props => (props.transparent ? "transparent" : "black")};
   border-bottom: 1px solid white;
-  ${props => props.transparent && !props.isFixed && `
+  ${props =>
+    props.transparent &&
+    !props.isFixed &&
```

Comment:
I'm assuming that this is suppose to be if there is a transparent property and `isFixed` is false apply those style rules?

Minor but I found this a bit tricky to read. I don't know if this is syntactically correct but maybe wrap the first two statements in parenthesis?

Reply:
Yep, this is syntactically correct. It's a little trick -- && will stop and return false at a false value, otherwise it will return the last value. So this returns the string literal if it passes the first two &&'s

Reply:
Right and what I was proposing was something like...

``` js
${props =>
  (props.transparent && !props.isFixed) &&
  `
  ...
`}
```

to group the two conditionals for readability.

Reply:
Oh gotcha, ya I'd be glad to do that!

Reply:
Ah well I tried to do this but Prettier just removed them anyway 😅

Reply:
Haha no worries. It was just a minor suggestion. In general this PR looks good to me. I think we can merge.